rotkehlchen/tests/utils/dataimport.py

```diff
@@ -1181,13 +1181,15 @@ def assert_bisq_trades_import_results(rotki: Rotkehlchen):
         base_asset=symbol_to_asset_or_token('BTC'),
         quote_asset=symbol_to_asset_or_token('EUR'),
         trade_type=TradeType.SELL,
-        amount=AssetAmount(FVal(2450)),
+        amount=AssetAmount(FVal(0.01)),
         rate=Price(FVal(49000.0000)),
         fee=Fee(FVal(0.29)),
         fee_currency=symbol_to_asset_or_token('BSQ'),
         link='',
         notes='ID: xxhee',
     )]
+    print(trades)
```

Comment:
Is this PR supposed to be ready? If yes wtf are 2 print statements doing here?

Reply:
My fault. Yes ready for review. Removed them